src/pages/workspace/reimburse/WorkspaceRateAndUnitPage.js

```diff
@@ -102,8 +101,6 @@ function WorkspaceRateAndUnitPage(props) {
             route={props.route}
             guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_REIMBURSE}
             shouldSkipVBBACall
-            backButtonRoute={ROUTES.WORKSPACE_REIMBURSE.getRoute(props.policy.id)}
-            shouldShowLoading={false}
```

Comment:
Revert?

Reply:
I've reverted the `shouldShowLoading={false}`, as the first line probably was removed intentionally (with this line back button on mobile doesn't work properly).